src/frontend/magnify/src/hooks/useFormState.ts

```diff
@@ -46,7 +46,7 @@ interface UseFormState<T> {
  *      As input, each validator receive the new value of the field, and the values of all other fields.
  * @returns UseFormState
  */
-export default function useFormState<T extends Record<string, string>>(
+export default function useFormState<T extends Record<string, string | boolean | number>>(
```

Comment:
Maybe this change should be reflected to the documentation above ?

Reply:
Ok. I'll look at it tomorrow. Thank you

Reply:
I'm unresolving this, just for not forgetting.

Reply:
I just fixed the wrong commit. This should be fixed now for the PR. The fixed commit is 18e15ca1